index.js

```diff
@@ -580,23 +577,35 @@ RingPop.prototype.onMemberSuspect = function onMemberSuspect(member) {
 
 RingPop.prototype.onMembershipUpdated = function onMembershipUpdated(updates) {
     var self = this;
+    var membershipChanged = false;
+    var ringChanged = false;
 
     updates.forEach(function(update) {
         if (update.type === 'alive') {
             self.onMemberAlive(update);
+            ringChanged |= membershipChanged |= true;
```

Comment:
true. it's a habit and since there's no one doing `|= false` the bit-wise operators are unnecessary. i'll take it out.